flow-decomposition/src/test/java/com/powsybl/flow_decomposition/FlowDecompositionResultsTests.java

```diff
@@ -95,7 +97,7 @@ void testBuilderNState() {
         assertEquals(branchId, decomposedFlow.getBranchId());
         assertEquals("", decomposedFlow.getContingencyId());
         assertEquals(branchId, decomposedFlow.getId());
-        assertEquals(10.0, decomposedFlow.getAcReferenceFlow());
+        assertEquals(10.0, decomposedFlow.getAcTerminal1ReferenceFlow());
```

Comment:
Can you add a test with terminal 2 as well please ?
`assertEquals(10.0, decomposedFlow.getAcTerminal2ReferenceFlow());`